docs/USAGE.md

```diff
@@ -0,0 +1,276 @@
+
+Personal API tokens provide many benefits, including isolation of security credentials between various usages and platforms. While a username and password are unique for a specific user, many security tokens can be attached to an individual user. If a token is considered compromised, that single token can easily be disabled or deleted to prevent its future use without impacting any platforms using other tokens generated for that user. Alternatively, dealing with compromised username and password credentials involves modifying those credentials on all multiple platforms.
+
+As a best practice, Jahia recommends that you create a separate token for each external service that needs to interact with the system APIs. This makes it easier for you to manage tokens that are compromised or lost.
+
+When created, a token is attached to its “creator” user and benefits from the same level of authorization as its creator. For this reason, treat tokens with the same level of precaution as traditional credentials.
+
+## Installing the module
+
+The personal API tokens module (module ID: personal-api-tokens) is compatible with Jahia 8.0.1.0 and above and can be easily installed by adding the module in Jahia Administration>Modules and Extensions>Modules. For information on adding the module, see the [Installing a module tutorial](https://academy.jahia.com/training-kb/tutorials/administrators/installing-a-module).
+
+Existing tokens are stored in the JCR and persist across Jahia upgrades as well as module upgrades.
+
+### Compatibility
+
+The Personal API tokens module is compatible with Jahia 8.0.1.0 with some limitations, and supports the entire feature set from Jahia 8.0.2.0. With Jahia 8.0.1.0, automatically deleting Groovy scripts when creating and managing tokens through the filesystem is not available. All other functionality, such as creating and using tokens through the UI or API, is supported.
+
+## Creating a token
+
+Tokens are active by default and can easily be disabled from the tokens list.
+
+:::info
+Access to the "My API tokens" page is restricted to users with permission "Personal api tokens" as a "Server role". Adding "Personal api tokens" permission to "Live role" or "Edit role" is not sufficient for a user to create and manage tokens.
+:::
+
+To create a token:
+
+1. In Jahia, navigate to **Dashboard>My API tokens**.
+2. Click **Create Token**.
+3. In the Create a personal API token dialog, provide the following information:
+      - **Name**  
+        The name of the token is unique for each user. It is used only for token management purposes to identify the token’s purpose by the user. 
+      - **Expiration date**  
+        The expiry date is the date after which any calls to the API are deemed invalid. Once the expiry date is reached, tokens are not deleted and are not usable unless you update the expiration date to a later date.  
+     
+     ![Create Personal Api Token 1](./img/create-personal-api-token-01.webp)
```

Comment:
Does that work on the academy ?

Why having both .png and .webp files in the repo ?